scaaml/capture/input_generators/attack_point_iterator.py

```diff
@@ -81,12 +77,30 @@ def _build_attack_points_iterator(configuration: Dict) -> AttackPointIterator:
 class AttackPointIteratorConstants(AttackPointIterator):
     """Attack point iterator class that iterates over a constant."""
 
-    def __init__(self, operation: str, name: str,
+    def __init__(self, operation: str, name: str, length: int,
                  values: List[List[int]]) -> None:
-        """Initialize the constants to iterate."""
+        """Initialize the constants to iterate.
+        
+            Args:
+                operation (str): The operation of the iterator
+                represents what the iterator does and what 
+                has to be in the config file. This is only used once to
```

Comment:
```suggestion
                arguments should be present. This is only used once to
```

Reply:
This class has no config file. It does not touch a config file. Not even the user-facing build_attack_points_iterator is concerned with files.